backend/compact-connect/lambdas/common-python/cc_common/data_model/schema/jurisdiction.py

```diff
@@ -60,7 +60,7 @@ class JurisdictionRecordSchema(BaseRecordSchema):
     @pre_dump
     def generate_pk_sk(self, in_data, **kwargs):  # noqa: ARG001 unused-argument
         in_data['pk'] = f'{in_data['compact']}#CONFIGURATION'
-        in_data['sk'] = f'{in_data['compact']}#JURISDICTION#{in_data['postalAbbreviation']}'
+        in_data['sk'] = f'{in_data['compact']}#JURISDICTION#{in_data['postalAbbreviation'].lower()}'
```

Comment:
Honestly yeah, but that would be a backwards incompatible change with what we currently have in the system. If we were to change it we would need to modify quite a few places in the code.

Reply:
Got it. Is there a reason to not just call the other field `compact` instead of `compactName`?

Reply:
Same reason, it's backwards incompatible to change it and would break the current system. @jusdino is planning on updating the compact config file schema in the `backend/compact-connect/compact-config` files to include the full compact name, at which point I believe the intent was to fix the compact schema here to use `compact` instead of `compactName` for that field.

Reply:
I agree the field name discrepancies should be fixed, but I don't believe that is a concern of this ticket, since this change is already not trivial and shouldn't mix two major schema changes in one PR. I can create a ticket on the backlog specifically for updating the compact/jurisdiction schemas to include the full compact names and fixing this issue.

Reply:
Ah ok. I misunderstood how easy/hard it'd be to change. All good! Thanks for the explanation.